frontend/src/v5/ui/routes/viewer/handleTicketsCardSearchParams/handleTicketsCardSearchParams.component.tsx

```diff
@@ -15,18 +15,26 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { DialogsActionsDispatchers, TicketsCardActionsDispatchers } from '@/v5/services/actionsDispatchers';
+import { DialogsActionsDispatchers, TicketsCardActionsDispatchers, ViewerGuiActionsDispatchers } from '@/v5/services/actionsDispatchers';
 import { TicketsHooksSelectors } from '@/v5/services/selectorsHooks';
 import { isEmpty } from 'lodash';
 import { useParams } from 'react-router-dom';
 import { useEffect } from 'react';
 import { formatMessage } from '@/v5/services/intl';
 import { ViewerParams } from '../../routes.constants';
 import { useSearchParam } from '../../useSearchParam';
+import { VIEWER_PANELS } from '@/v4/constants/viewerGui';
 
-export const OpenTicketFromUrl = () => {
-	const [ticketId, setTicketId] = useSearchParam('ticketId');
+export const HandleTicketsCardSearchParams = () => {
 	const { containerOrFederation } = useParams<ViewerParams>();
+	const [ticketId, setTicketId] = useSearchParam('ticketId');
+
+	const [queriesRaw, setQueriesParam] = useSearchParam('queries');
+	const queries = queriesRaw?.split(',').filter((f) => f);
```

Comment:
If I understand correctly instead of doing this , you could do this
```suggestion
	const queries = const [modelsIds] = useSearchParam('queries', Transformers.STRING_ARRAY);
```
like in https://github.com/3drepo/3drepo.io/blob/5f3dccf3a2fb4e0225c301ea8c41ada6eb2eb698/frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/newTicketMenu/useSelectedModels.ts#L24

Reply:
I didn't know about this. Useful!